tests/testthat/test-simple.R

```diff
@@ -30,22 +30,32 @@ if (!testthat:::on_cran()) {
     expect_warning(tsmp(data, data, data, window_size = w, verbose = 0, mode = "simple"), "Only the first two")
   })
 
-  if (!testthat:::on_cran()) {
-    result_self <- simple_fast(list(data[, 1], data[, 2], data[, 3]), window_size = w, verbose = 2)
-    result_join <- simple_fast(as.data.frame(t(data)), as.data.frame(t(query)), window_size = w, verbose = 2)
-  } else {
-    result_self <- simple_fast(data, window_size = w, verbose = 0)
-    result_join <- simple_fast(data, query, window_size = w, verbose = 0)
```

Comment:
This branch is never called because this itself is scoped within an `!testthat:::on_cran()` if-block.